config.yaml

```diff
@@ -0,0 +1,8 @@
+netbox:
```

Comment:
this config should only contain placeholder. we fill it in helm-charts

Reply:
Done.